### PR TITLE
Update cronjob to deploy testing clusters to kubeflow-ci-deployment

### LINKED ISSUE #350: Update auto-deploy for v0-5
Currently both `kf-vmaster` and `kf-v0.4` are not deployed properly and I think the reason is they are still using shell script of `kfctl`.  Need to switch to go binary for deployment.

### PR DESCRIPTION
**Pull request by gabrielwen**
Fixes #350
Updates with this PR:
1. Adds back labels to clusters.
1. Use `CLIENT_ID/CLIENT_SECRET` from kubeflow-ci-deployment.
1. Fixes base name for v0-5 cronjobs.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/testing/352)

### CODE REVIEW

**Comment by gabrielwen** [comment]
Example deployments are succeeded:
- http://kf-vmaster-n00.endpoints.kubeflow-ci-deployment.cloud.goog
- http://kf-vmaster-n01.endpoints.kubeflow-ci-deployment.cloud.goog
- http://kf-vmaster-n02.endpoints.kubeflow-ci-deployment.cloud.goog
- http://kf-v0-5-n01.endpoints.kubeflow-ci-deployment.cloud.goog

**note**: kf-v0-5-n00 is brought down because of kubeflow/kubeflow#2550

File: test-infra/auto-deploy/deploy-cron-master.yaml [hunk 1]
```diff
@@ -14,14 +14,18 @@ spec:
         spec:
           containers:
           - name: deploy-worker
-            image: gcr.io/kubeflow-ci/deploy-worker:v20190302-afc8ef8-dirty-201fe5
+            image: gcr.io/kubeflow-ci/deploy-worker:v20190403-33a1352-e3b0c4
             env:
             - name: GOOGLE_APPLICATION_CREDENTIALS
               value: /secret/gcp-credentials/key.json
+            - name: CLIENT_ID
```
Comment (jlewi) [inline test-infra/auto-deploy/deploy-cron-master.yaml]: Can we load CLIENT_ID and CLIENT_SECRET from an OAuth secret as opposed to hard  coding the values?

How are we doing this right now? Aren't we loading the keys from a GCS file?

Comment (gabrielwen) [reply test-infra/auto-deploy/deploy-cron-master.yaml]: done.

**Comment by jlewi** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**